docs/Windows-DEVELOP.md

```diff
@@ -0,0 +1,112 @@
+# Windows Development Guide for rdiff-backup
+
+Some notes for developers and other people willing to help development on
+Windows platform, or simply compile rdiff-backup from source code on Windows.
+
+See also: https://wiki.python.org/moin/WindowsCompilers
+
+## Pre-requisites
+
+Here the list of required component to be install to start developing
+rdiff-backup on Windows platform.
+
+### Upgrade your Windows OS
+Don't overlook this step. Visual Studio required an upgraded OS.
+Use Windows Update to upgrade your system. Then proceed with the installation.
+
+### Visual Studio Build Tools
+
+1. To install Visual Studio Build Tools, you need to install .NET Framework 4.6.  
+   [Download .NET Framework 4.6](https://www.microsoft.com/en-US/download/details.aspx?id=53344)
+2. Finally, download Build Tools installation.  
+   [Microsoft Build Tools for Visual Studio 2019](https://www.visualstudio.com/downloads/#build-tools-for-visual-studio-2019)
+3. Once downloaded, the filename should be `vs_buildtools.exe` or similar, launch it to start the installation.
+4. From the user interface select "C++ Build Tools"
+5. In the right panel checked the following items. Everything else may be unchecked.
+    * MSVCv142 - VS 2019 C++ x64/x86 build tools
+    * Windows 10 SDK
+    * C++ CMake Tools for Windows
+
+### Python 3
+
+[Download Python 3.7.5 x86](https://www.python.org/ftp/python/3.7.3/python-3.7.3-amd64.exe)
+
+1. Check "Add Python 2.7 to PATH"
+2. Choose "Customize instalation" to select the installation path.
+3. Check "Install for all users". That should install Python to `C:\Program Files (x86)\Python37-32`
+
+You could verify if Python is working by executing `python --version` in a new command line windows.
+
+NOTICE: Python 3.7.3 is not supported because it failed to compile external libraries.
+
+### Python dependencies
+
+Once python is installed, you should have a `pip` available from command line.
+Open a terminal and execut the following commands to install the dependencies required to compile and run rdiff-backup.
+
+    pip install pywin32
+

```

Comment:
I think adding that `python.exe -c 'import pywintypes, winnt, win32api, win32security, win32file, win32con'` should be successful at this point is a good check.